src/scep/Client/certificate.py

```diff
@@ -86,7 +86,8 @@ def end_date(self):
     def key_usage(self):
         if self._certificate.key_usage_value is not None:
```

Comment:
I just tested it, it worked. This code is from upstream master that wasn't up to date on our branch, but the upstream code works.

Reply:
where is the code from upstream master? I couldn't find the PR from upstream that changed it.

Reply:
My bad, this is from our repo: https://github.com/foxpass/PyScep/commit/7823427bf8c3ae5c19b14f3c302e617bd4af463a (Bryan's commit). On the ec2 machine we used PyScep directly and made the change manually to return an array instead of this fix.